documentation/airport-guides/loww.md

```diff
@@ -0,0 +1,77 @@
+### Vienna Schwechat International Airport
+Vienna (Wien) Schwechat International Airport is the largest airport in Austria, serving the Austrian capital Vienna. It is located in the town of Schwechat, 11 miles southeast of Vienna. The airport serves as a base for Austrian Airlines and Eurowings Europe, and is a focus city for Lauda (formerly known as Laudamotion), Anisec Luftfahrt (branded as LEVEL) and Wizz Air.
+
+The Airport was originally built in 1938 as a military design and production complex. There was just one runway (11/29), which was expanded in 1959, and a second runway (16/34) was built in 1972. In 2006 a new control tower was opened which allows for overview of the entire airport.
+
+### Statistics for LOWW, 2018
+27.037.292 passengers
+295.427 tonnes of cargo
+241.004 aircraft movements
+
+### Airspace
+The Vienna approach is sourrounded by the FIRs of Vienna (LOVV), Praha (LKAA), Bratislava (LZBB) and Budapest (LHCC).
+
+The Airspace located within the LOVV FIR is controlled from SFC to FL245. Wien Radar also controls part of the Budapest FIR, southeast of Vienna. The airspace in this area is controlled by Wien Radar from FL055 to FL245.
+
+### Restricted Airspace
+Aircraft departing from runway 29 make a left turn at 1.000 feet due to restricted airspace LOR-15, which lies over the city of Vienna, and extends up to FL100.
+
+Aircraft arriving into Vienna cross through restricted airspace LOR-15 when approaching runway 11 and 16. IFR flights are allowed to enter this airspace when approaching the airport eastbound or southbound. There are other exceptions to this restriction, including VFR flights in accordance to prescribed procedures or military operations.
+
+### Instrument Procedures
+
+#### Departures
+Aircraft taking off from Vienna follow RNAV SIDs. These SIDs have suffixes corresponding to certain runways.
+
+| Suffix | Runway          |
+|:------:|:---------------:|
+| A      | 11              |
+| B      | 16              |
+| C      | 29              |
+| D      | 34              |
+| R      | Noise Abatement |
+
+#### Arrivals
+Aircraft descending into Vienna are funneled onto the ILS by RNAV transitions. Each transition has a certain suffix corresponding to the respective runway.
```

Comment:
Change to:
`Each transition has a suffix that corresponds to a specific runway`